doc/man1/s_server.pod

```diff
@@ -710,13 +711,13 @@ greater than or equal to 0.
 
 Accept early data where possible.
 
-=item B<-no_anti_replay>
+=item B<-anti_replay>, B<-no_anti_replay>
 
-Switches off replay protection. With replay protection on OpenSSL will
-automatically detect if a session ticket has been used more than once, TLSv1.3
-has been negotiated, and early data is enabled on the server. A full handshake
-is forced if a session ticket is used a second or subsequent time. Any early
-data that was sent will be rejected.
+Switches on or off replay protection respectively. With replay protection on
+(the default), OpenSSL will automatically detect if a session ticket has been
```

Comment:
Do you think it might be appropriate to instead say something along the lines of "(compiled-in default, also supports configuration file overrides)" to indicate why we need to have a switch that forces a default?  Not critical, but perhaps makes the purpose more obvious.